inst/templates/head.html

```diff
@@ -4,7 +4,13 @@
 
 <title>{{{pagetitle}}} • {{#site}}{{title}}{{/site}}</title>
 
-<link rel="shortcut icon" type="image/x-icon" href="{{#site}}{{root}}{{/site}}favicon.ico">
+<link rel="icon" type="image/x-icon" href="{{#site}}{{root}}{{/site}}favicon.ico">
```

Comment:
Actually, some sites suggest **not** to declare the `.ico` file. One of the reason the footnote [2] [here](https://caniuse.com/#search=favicon). Should I drop it? If it's a the root of the website, IE will find it anyways, even if we don't declare it.

Reply:
I went ahead and removed it. I think that's the right choice.

Reply:
It will only break stuff for the [0.37% IE users](https://caniuse.com/#search=favicon), when the `pkgdown` website is not at the root of the website.

Reply:
Yeah, let's drop it.